### DISCUSSION TITLE
Update Configuration Only

### DISCUSSION THREAD

**Discussion by jamos104** [tijsverkoyen/HomeAssistant-FusionSolar#152] [General]
After downloading the Fusion Solar add-in I didn't get the confirmation flow. I only have the Update configuration. Should I delete remove and try again? I can't see how to configure my kiosk URL

**Comment by tijsverkoyen** [comment]
Do you have a screenshot or something? Because I don't understand.

**Comment by jamos104** [comment]
Hi,

Yes, during install I did not get the opportunity to put in my Kiosk address. I only have the update services available. Maybe I misread the install procedure. Please bare in mind this is my first HA integration...
<img width="754" alt="Screenshot 2024-02-21 143609" src="https://github.com/tijsverkoyen/HomeAssistant-FusionSolar/assets/79424465/62b0aa50-edfa-44df-861a-37eaab383c29">
<img width="319" alt="Screenshot 2024-02-21 143914" src="https://github.com/tijsverkoyen/HomeAssistant-FusionSolar/assets/79424465/526a6503-6abd-474f-9095-e0fea1212887">
<img width="754" alt="Screenshot 2024-02-21 143609" src="https://github.com/tijsverkoyen/HomeAssistant-FusionSolar/assets/79424465/19c47191-1c4c-42ed-ad82-fa6e1e28b450">

**Comment by tijsverkoyen** [comment]
Can you remove the integration. Restart Home Assistant. Add the integration again.

**Reply by jamos104** [reply]
I removed the download. Rebooted the HA. Then I hit download, steps 1 and 2 of the picture below. Restarted after that. Same issue. I have also attached a log.

Thanks. Andrew
[config_entry-hacs-a7db3248bfb53fdc08517aed67ec379f.json](https://github.com/tijsverkoyen/HomeAssistant-FusionSolar/files/14357279/config_entry-hacs-a7db3248bfb53fdc08517aed67ec379f.json)
![Install 1](https://github.com/tijsverkoyen/HomeAssistant-FusionSolar/assets/79424465/e78e7eb9-9525-4f18-86a0-e889e9e1a51c)

**Comment by tijsverkoyen** [comment]
And have you added the integration afterwards? 
Settings → Devices & Services → Integrations → Add integration. Search for "Fusion Solar"

**Reply by jamos104** [reply]
I have now, sorry, as I said noob. I sent you a donation for a few beers.

It is now showing 5 sensors, all to do with electricity generation, using Kiosk mode. Is it possible to get how much is being put back in the grid and what is being drawn from the grid?

API method seems to have that type of info.

**Reply by tijsverkoyen** [reply]
> I have now, sorry, as I said noob. I sent you a donation for a few beers.
Really appreciated!

> It is now showing 5 sensors, all to do with electricity generation, using Kiosk mode. Is it possible to get how much is being put back in the grid and what is being drawn from the grid?
> API method seems to have that type of info.

No. The kiosk data I retrieve does not expose this.
